components/backup/src/endpoint.rs

```diff
@@ -452,10 +501,20 @@ impl BackupRange {
         let mut batch = vec![];
         loop {
             while cursor.valid()? && batch.len() < BACKUP_BATCH_LIMIT {
-                batch.push(Ok((
-                    cursor.key(cfstatistics).to_owned(),
-                    cursor.value(cfstatistics).to_owned(),
-                )));
+                let key = cursor.key(cfstatistics);
```

Comment:
Can we reuse `RawStore::forward_raw_scan` here ?

Reply:
RawStore  return the value without ttl info, we need keep ttl. So here just RawMvccSnapshot is reused.